docs/_data/related.csv

```diff
@@ -0,0 +1,3 @@
+rh-button, rh-cta
```

Comment:
I had considered this, but relationships are by groups of elements not individual ones, so we'd have to maintain the same sets of relationships across multiple pages. For example, instead of having a single row of navigation elements in a centralized location, we'd have to add go to each navigational element and add all of the other nav elements to it's related items section. It just seemed redundant and like it would invite the opportunity for errors and inconsistencies.

Reply:
which would @marionnegp and @coreyvickery find easier to maintain: a central file listing all relationships, or for each file, listing the related links in header.

I think a new contributor (or a current contributor that forgets we had this conversation in a few months) might forget easily that all the relationships are centrally stored.

If we do go for a central store, can it be a yaml file instead, to cut down on manual parsing?

Reply:
@nikkimk Need you to respond to Benny first I think before I can add my review.

Reply:
Correction: It turns out we will need it in a shared spot as each element's related items will be repeated across every on of its docs pages. See [Corey's](https://github.com/RedHat-UX/red-hat-design-system/pull/942#issuecomment-1533437653) comment below.